validation/standard/standard-versions.toml

```diff
@@ -1,14 +1,38 @@
-["op-contracts/v1.4.0"] # Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
-address_manager = ""
-l1_cross_domain_messenger = "2.3.0"
-l1_erc721_bridge = "2.1.0"
-l1_standard_bridge = "2.1.0"
-optimism_mintable_erc20_factory = "1.9.0"
-optimism_portal = "3.10.0"
-system_config = "2.2.0"
-protocol_versions = "1.0.0"
-anchor_state_registry = "1.0.0"
-delayed_weth = "1.0.0"
-dispute_game_factory = "1.0.0"
-fault_dispute_game = "1.2.0"
-permissioned_dispute_game = "1.2.0"
+op_contracts_release = "op-contracts/v1.4.0"
+
+[contract_tag]
+
+# Fault Proofs https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.4.0
+[contract_tag."op-contracts/v1.4.0"]
+address_manager = { version = "" }
+l1_cross_domain_messenger = { version = "2.3.0" }
+l1_erc721_bridge = { version = "2.1.0" }
+l1_standard_bridge = { version = "2.1.0" }
+optimism_mintable_erc20_factory = { version = "1.9.0" }
```

Comment:
Looks like a lot of addresses are missing here:
- [Here](https://www.notion.so/oplabs/reMCP-L1-Mainnet-7e216b38c06c48298fe7b38d7dbbf952?pvs=4) is where you can find 1.3.0 addresses
- [Here](https://www.notion.so/oplabs/Fault-Proofs-Mainnet-Deployment-041500821d614593b8565116ebc06f50?pvs=4) links to the 1.4.0 info, but I'd suggest confirming with proofs team
- [This](https://www.notion.so/oplabs/Mainnet-FP-Fixes-Deployment-8f641ce6d3094a5f8c6105184f43781b?pvs=4) should have the info for 1.6.0, again I'd suggest confirming with proofs team

Reply:
Thanks @mds1 . Received confirmation from @Inphi on Discord that these are indeed the addresses.